README.md

```diff
@@ -121,9 +121,15 @@ Coming soon
   - Login with `admin` and the result from step 4
 
 ### Upgrading Ingest Monitoring Dashboard
```

Comment:
Why is the **monitoring** section under the U**pgrade Backend Services** section?